src/terra-dev-site/doc/application/reference.1/components/ApplicationNavigation.doc.mdx

```diff
@@ -0,0 +1,46 @@
+import PropsTable from 'terra-application/lib/application-navigation/ApplicationNavigation?dev-site-props-table';
+import ApplicationNavigationExample from 'terra-application/lib/terra-dev-site/doc/application/reference.1/components/example/ApplicationNavigationExample?dev-site-example';
+
+# ApplicationNavigation
+
+The ApplicationNavigation component provides a responsive layout and a variety of controls for application navigation.
+
+## Usage
+
+```jsx
+import ApplicationNavigation from 'terra-application/lib/application-navigation';
+```
+
+## Props
+
+<PropsTable />
+
+## Example
+
+<ApplicationNavigationExample />
+
+## Features
```

Comment:
Looking at the deployed site, I think it'd be nice to read the features before the prop types

Reply:
I waffled a bit on this. I originally had things ordered the same way, but anyone using the doc site as a prop reference would be scrolling around a lot. Figured for the reference pages, the props might be more ideal?